lib/Conductor.js

```diff
@@ -153,6 +152,18 @@ function Conductor(config) {
         models: config.models || {}
     };
 
+    // If props isn't a function and is instead an object, wrap it
+    if (_.isPlainObject(this._config.propsFn) &&
```

Comment:
Wonder if it might be easier to just move all the `propsFn` initialization into a single place here so it's easier to follow? You could just initialize it to null and keep the jsdoc.